scripts/run.sh

```diff
@@ -26,243 +18,80 @@ source "$SUBNET_EVM_PATH"/scripts/versions.sh
+
+mkdir -p $DATA_DIR
+
+# Set the config file contents for the path passed in as the first argument
+function _set_config(){
+  cat <<EOF >$1
+  {
+    "network-id": "local",
+    "staking-enabled": false,
+    "health-check-frequency": "5s",
+    "plugin-dir": "$AVALANCHEGO_PLUGIN_DIR"
+  }
 EOF
+}
 
-#################################
-# download avalanche-network-runner
-# https://github.com/ava-labs/avalanche-network-runner
-ANR_REPO_PATH=github.com/ava-labs/avalanche-network-runner
-# version set
-go install -v ${ANR_REPO_PATH}@${ANR_VERSION}
-
-#################################
-# run "avalanche-network-runner" server
-GOPATH=$(go env GOPATH)
-if [[ -z ${GOBIN+x} ]]; then
-  # no gobin set
-  BIN=${GOPATH}/bin/avalanche-network-runner
-else
-  # gobin set
-  BIN=${GOBIN}/avalanche-network-runner
-fi
-echo "launch avalanche-network-runner in the background"
-$BIN server \
-  --log-level debug \
-  --port=":12342" \
-  --grpc-gateway-port=":12343" &
-PID=${!}
+DATA_DIRS=()
+CMDS=()
+for (( i=0; i <5; i++ ))
+do
+  NODE_NAME=node$(($i+1))
+  NODE_DATA_DIR="$DATA_DIR/$NODE_NAME"
+  DATA_DIRS+=("$NODE_DATA_DIR")
+  echo "Creating data directory: $NODE_DATA_DIR"
+  mkdir -p $NODE_DATA_DIR
+  NODE_CONFIG_FILE_PATH="$NODE_DATA_DIR/config.json"
+  _set_config $NODE_CONFIG_FILE_PATH
+  
+  CMD="$AVALANCHEGO_PATH --data-dir=$NODE_DATA_DIR --config-file=$NODE_CONFIG_FILE_PATH"
+  if [ $i -gt 0 ]; then
+    echo "Adding CLI options for node$(($i+1))"
+    CMD="$CMD --staking-port=$((9651+2*$i)) --http-port=$((9650+2*$i)) --bootstrap-ips=127.0.0.1:9651 --bootstrap-ids=NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg"
```

Comment:
could we define the expected nodeID with the certs in a variable (if we can't generate keys and infer it that way?)

Reply:
The goal here is that the key remains constant, so that we can easily populate the bootstrap ID. This seemed like the easiest way to do that.

What would you suggest we change it to?

Reply:
Removing this in favor of keeping this script simple to run a single non-staking node. We can use other tools to do more complex tasks.